### PR TITLE
[New] Add jquery.threedubmedia@2.2 w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #
Related issue(s): #10036 #10444

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/threedubmedia/jquery.threedubmedia
 * Official website (optional, not the repository): http://threedubmedia.com
 * NPM package url (optional):
 * License and its reference: [MIT](http://threedubmedia.com/code/license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 12
   - Count of stars: 114
   - Count of forks: 67
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3 @dakshshah96 
Please help me review this PR, Thank you.

Note: 
I follow the [homepage](http://threedubmedia.com/) just added `jQuery.event.drag` and `jQuery.event.drop` part.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @cdnjs/library-reviewer 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@pvnr0082t I modified the commit message.
Please help me review this PR, Thank you.

**Comment by maruilian11** [comment]
Since the homepage only mention `jQuery.event.drag` and `jQuery.event.drop`, I think we should still host all product on the repo. And the repo has enough popularity.

**Comment by sufuf3** [comment]
@maruilian11 I added all products on the repo except jquery.
Please help me review this PR, Thank you.

File: ajax/libs/jquery.threedubmedia/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "homepage": "http://threedubmedia.com/",
+  "author": "threedubmedia",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/threedubmedia/jquery.threedubmedia.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "+(event*|fire)/+(jquery.event.*|jquery.fire*)"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.threedubmedia/package.json]: Can we just use `"!(jquery)/+(jquery.*)"`?

**Comment by sufuf3** [comment]
@dakshshah96 @cdnjs/intern3 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@cdnjs/intern3 @cdnjs/library-reviewer 
Please help me review this PR, Thank you.

**Pull request merged**